### DISCUSSION TITLE
french language problem

### DISCUSSION THREAD

**Discussion by plabrie19** [williamtroup/Calendar.js#250] [General]
Hello,
I noticed some translation and layout issues in the extension. see attach files

Thanks

![imgparaprob](https://github.com/user-attachments/assets/5af62c74-5df2-427b-a361-b9e2bc236c23)

**Comment by william-troup** [comment]
Hello,

In the first screenshot, you can set translations for these yourself using the public function "addEventType()".

In the last two screenshots, the checkboxes are centered because something in your CSS overrides the container.  You can set the "text-align: left" using an override for the container to fix the issue.

I hope this helps.

Will Troup
